hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/record/AbstractReplicatedRecordStore.java

```diff
@@ -289,18 +289,12 @@ public int size() {
 
     @Override
     public void clear() {
-        InternalReplicatedMapStorage<K, V> storage = getStorage();
-        storage.clear();
-        storage.incrementVersion();
-        getStats().incrementOtherOperations();
+        clearInternal();
```

Comment:
Hmm, is this missing the `.incrementVersion()` call?

Reply:
oops! nice catch. fixed.